File: AOJ/0370/age_difference.cpp
#include<iostream>
using namespace std;

int main() {
  pair<int, pair<int,int>> p1, p2;

  cin >> p1.first >> p1.second.first
      >> p1.second.second;
  cin >> p2.first >> p2.second.first
      >> p2.second.second;

  if ( p1 > p2 ) swap(p1, p2);

  int ans = p2.first - p1.first;
  if (p1.second < p2.second) ans++;
  cout << ans << endl;
    
  return 0;
}

File: AOJ/0370/age_difference.py
y1, m1, d1 = map(int, input().split())
y2, m2, d2 = map(int, input().split())

if y1 > y2 or (y1 == y2 and (m1 > m2 or ( m1 == m2 and d1 > d2 ))):
    y1, y2 = y2, y1
    m1, m2 = m2, m1
    d1, d2 = d2, d1
ans = y2 - y1
if m1 < m2 or (m1 == m2 and d1 < d2): ans += 1
print(y2 - y1)

